### PR TITLE
Add support for Eltako FSR61 switches (Enocean)

### PR DESCRIPTION
**Pull request by rubund**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.enocean.markdown [hunk 1]
```diff
@@ -13,19 +13,25 @@ ha_release: 0.21
...
 
+
 ```yaml
 # Example configuration.yaml entry
 switch:
   - platform: enocean
+    name: my_switch_name
     id: [0x01,0x90,0x84,0x3C]
+    sender_id: [0xff,0xc6,0xea,0x14]
+    subtype: fsr61
```
Comment (fabaff) [inline source/_components/switch.enocean.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by fabaff — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]
@rubund @fabaff Updated PR. Some small fixes and processed review comment.

Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Rebased the PR in order to fix the build

**Review by frenck — APPROVED** [review]

**Comment by DubhAd** [comment]
I note that the linked PR was closed 14 days ago, closing this too

**Pull request closed without merging**